src/main/webapp/referral-X-workz-companies-emailId.html

```diff
@@ -1,273 +1,273 @@
-							placeholder="Enter Number of Position">
-					</div>
-					<div class="col-md-6 contact-fields">
-						<input type="text" class="technicalSkill" name="technicalSkill"
-							id="technicalSkill" placeholder="Enter Technical Skill"
-							required="">
-					</div>
-
-					<div class="col-md-12 contact-fields">
-						<textarea name="comments" id="otherComment" rows="4"
-							placeholder="Any mobile number for refferal,empid if required"></textarea>
-					</div>
-
-					<button id="refSubmit" type="submit"
-						class="btn btn-lg btn-block referral submitReff">
-						<b>Submit</b>
-					</button>
-					<!-- <input type="submit" value="submit" class="btn btn-primary btn-lg btn-block invisible"> -->
-					<!-- <div class="mt-2"></div> -->
-				</form>
-			</div>
-		</div>
-		<div class="footer-agileits-w3layouts">
-			<div class="container">
-				<div class="btm-logo-w3ls">
-					<h2>
-						<a href="index.html"> <img src="assets/images/Logo.png"
-							height="100px" width="220px" />
-						</a>
-					</h2>
-				</div>
-				<div class="subscribe-w3ls">
-					<h6>Let us inform you about everything important directly.</h6>
-					<form action="#" method="post">
-						<div class="col-md-8 col-sm-8 col-xs-8 input-flds-w3-agile">
-							<input type="email" name="Email" placeholder="Email" required=""
-								id="emailSubField">
-						</div>
-						<div class="col-md-4 col-sm-4 col-xs-4 input-flds-w3-agile">
-							<input type="button" value="Subscribe" id="subscribe"
-								class="subscribeForm">
-							<button type="submit" hidden="true" id="submit">submit</button>
-						</div>
-						<div class="clearfix"></div>
-					</form>
-				</div>
-				<div class="social-icons-agileits">
-					<ul>
-						<li><a
-							href="https://www.facebook.com/xworkzdevelopmentcenter/"><span
-								class="fa fa-facebook"></span></a></li>
-						<li><a href="https://twitter.com/workz_x"><span
-								class="fa fa-twitter"></span></a></li>
-					</ul>
-				</div>
-
-				<div class="clearfix"></div>
-			</div>
-		</div>
-		<div class="copyright-w3layouts">
-			<div class="container">
-				<p>&copy; 2019 X-Workz . All Rights Reserved</p>
-			</div>
-		</div>
-
-	</div>
-	<a href="#home" class="scroll" id="toTop" style="display: block;">
-		<span id="toTopHover" style="opacity: 1;"> </span>
-	</a>
-
-<script type="text/javascript" src="js/mainJS.js"></script>
-
-</body>
-
-
-</html>
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+<title>X-workz-referral interviews</title>
+<link rel="shortcut icon" href="assets/images/Logo.png" />
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+<meta name="keywords"
+	content="The 10 Best Tech Jobs,BE jobs, Software Developer, Data Scientist, Information Security Analyst, Computer Systems Analyst, Web Developer, Sales Engineer,  Information Technology Manager, Computer Research Scientist,
```

Comment:
we dont train on these....use only which is preferred

Reply:
I tried to give different meta-info on each page sir